ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -184,7 +219,15 @@ - (void)startLoading
 
 - (void)stopLoading
 {
-    AD_LOG_VERBOSE(_reqContext(self.request).correlationId, @"%@ - host: %@", @"-[ADURLProtocol stopLoading]", [self.request.URL host]);
+    if ([ADAuthorityUtils isKnownHost:self.request.URL])
+    {
+        AD_LOG_VERBOSE(_reqContext(self.request).correlationId, @"%@ - host: %@", @"-[ADURLProtocol stopLoading]", [self.request.URL host]);
+    }
+    else
+    {
+        AD_LOG_VERBOSE(_reqContext(self.request).correlationId, @"-[ADURLProtocol stopLoading]");
+        AD_LOG_VERBOSE_PII(_reqContext(self.request).correlationId, @"%@ - host: %@", @"-[ADURLProtocol stopLoading]", [self.request.URL host]);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 144 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)